lib/bump/version_control.rb

```diff
@@ -0,0 +1,44 @@
+module Bump
+  class VersionControlNotFoundError < StandardError; end
+
+  class VersionControl
+    def self.git?
```

Comment:
Did not think there was any harm in giving access to all methods as they might be useful. And the one not currently not used outside does not change anything (git?, mercurial?, commit_message).

Will make them private if you wish so.